snare/utils/snare_helpers.py

```diff
@@ -33,11 +40,17 @@ def check_compatibility(self, tanner_version):
 
 
 class Converter:
-    def __init__(self):
+    def __init__(self) -> None:
+        """Converter class"""
         self.logger = logging.getLogger(__name__)
         self.meta = {}
 
-    def convert(self, path):
+    def convert(self, path: str) -> None:
+        """Convert all pages to a Snare-friendly form and write meta info
```

Comment:
what is Snare-friendly format? :)

Reply:
I meant to convey the renaming of files to their hashes but I realize it sounds very vague. I have clarified it in 59dd4d2.